src/main/java/org/jboss/tools/intellij/openshift/actions/cluster/LoginAction.java

```diff
@@ -33,9 +36,10 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
             return dialog;
             });
           if (loginDialog.isOK()) {
+            ExecHelper.setProxyEnvironmentVariables(OkHttpClientProxy.buildEnvironmentVariables(loginDialog.getClusterURL()));
```

Comment:
No that should not be managed externally like that. The tree model has an refresh method that is called when the context is changed, that should be the starting point.

Reply:
👍